runtime/vm/jvminit.c

```diff
@@ -2625,36 +2625,38 @@ modifyDllLoadTable(J9JavaVM * vm, J9Pool* loadTable, J9VMInitArgs* j9vm_args)
 		}
 #endif
 
+		uintptr_t argFlag = ((NOT_CONSUMABLE_ARG) | ((j9vm_args->j9Options[i].flags) & (USER_ARG)));	
```

Comment:
same here with the extra brackets

Reply:
I get an error when I remove the brackets around `((j9vm_args->j9Options[i].flags) & (USER_ARG)))`, so atm, I'll just remove the brackets around `NOT_CONSUMABLE_ARG`

Reply:
what kind of error?

Reply:
it didn't like how there was a `|` and a `&` in the same expression, and asked me to brackets for the & case

So it ended up looking like: 

```uintptr_t argFlag = NOT_CONSUMABLE_ARG | (j9vm_args->j9Options[i].flags & (USER_ARG));	```